card-flow.md

```diff
@@ -0,0 +1,52 @@
+# Project Management Boards
+
+Our team uses project boards to manage our projects and track our progress.  The boards serve two purposes: to allow clients to see what's happening and give feedback on a project, and to allow us to manage our team, our tasks, and stay coordinated. Each project has its own board. The board shows us the current state of all the things we're working on in our project and where each task is in the lifecycle of development. Each task is represented by a card on the board, and as cards move through the development process, we move them to different columns on the board.
+
+Everything the team is working on for a project, whether a feature, a bug, or a chore, should be represented by a card. We can't track something if it's not on the board. If we're doing something that's not represented by a card, we should ask why.
+
+## Board Flow
+
+Our current project board tool is not very expressive in some ways, and this leads us to add additional columns to our board. Currently, we have a "Backlog" column in our project board. Once Feature Complete is up and running, our backlog will live in the Features area of the app rather than on the project board itself. The "Backlog" column is where we store tasks that will need to be completed but it's either not the right time to begin work on them or they haven't yet been approved.
+
+#### The "Next" Column
+Once cards are approved and ready for developers to begin working on them, we move them to the "Next" column. Cards in this column are our queue of work, and are waiting for someone to be assigned to them. A developer can assign themselves to a card when they are ready to begin work on it.
+
+#### The "Development" Column
+
+The "Development" column is for cards that are actively being worked on. It is important that a developer has only one card in this column at a time. If they need to work on something else, this card can move back to ready so as not to let things build up or block. Our project board system follows the traditional Kanban pull based system pretty closely. Things can only move right if there's space for them, and a dev team can really only take on 3-4 cards at once in the development stage. More than that and things sit idle. The board should be an accurate reflection of the project at any point in time, so if a developer is not currently working on a card in the "Development" column, they can move it back so our progress is accurately reflected.
+
+#### The "Needs Review" Column
+
+When a developer has completed the task on a card, we kick off the (pull request process)[https://github.com/HaughtCodeworks/TeamGuide]. The "Needs Review" column represents cards currently under code review in the PR process. After a PR has been fully approved, the feature is merged into the master branch and deployed to the staging environment.
```

Comment:
This link is just a placeholder for now, until all the documents are in the  master branch.